static/js/modules/cycle-select.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/* global window */
+
+var $ = require('jquery');
+var _ = require('underscore');
+var URI = require('URIjs');
+
+function CycleSelect(elm) {
+  this.$elm = $(elm);
+  this.$elm.on('change', this.handleChange.bind(this));
+}
+
+CycleSelect.build = function($elm) {
+  switch ($elm.data('cycle-location')) {
+  case 'query':
```

Comment:
I've seen both styles--for example, https://contribute.jquery.org/style-guide/js/#switch-statements. But now that I look at style recommendations, I also see that folks generally don't like `switch` statements, since they're easy to mess up. I'll just use a regular conditional here.